### PR TITLE
chore(gatsby-source-contentful): drop last usages of lodash

### PR DESCRIPTION
**Pull request by pvdz**
Drops the last remaining cases of lodash from this file.

See inline comments for clarification

### CODE REVIEW

File: packages/gatsby-source-contentful/src/extend-node-type.js [hunk 1]
```diff
@@ -178,11 +176,8 @@ const resolveFixed = (image, options) => {
     )
   })
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
```
Comment (pvdz) [inline packages/gatsby-source-contentful/src/extend-node-type.js:182]: The array is initialized ordered. It is only pruned, not shuffled / added. So this sort should be a noop in any case.

File: packages/gatsby-source-contentful/src/extend-node-type.js [hunk 2]
```diff
@@ -286,19 +281,17 @@ const resolveFluid = (image, options) => {
...
+    !filteredSizes.includes(parseInt(pwidth, 10)) &&
+    pwidth < CONTENTFUL_IMAGE_MAX_SIZE &&
+    Math.round(pwidth / desiredAspectRatio) < CONTENTFUL_IMAGE_MAX_SIZE
   ) {
-    filteredSizes.push(width)
+    filteredSizes.push(pwidth)
   }
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
```
Comment (pvdz) [inline packages/gatsby-source-contentful/src/extend-node-type.js:298]: This sort is similar to above except the condition before it does not exist in the other case.

The condition will only add the current width if it doesn't already exist here. And the loop before will cull any widths that are greater than this width. So either way, the width should be the last element in the list. This means the `.sortBy` is equally useless here and is safe to drop.

File: packages/gatsby-source-contentful/src/extend-node-type.js (hunk 1 above)
Comment (vladar) [reply packages/gatsby-source-contentful/src/extend-node-type.js:182]: This warrants a comment that `fixedSizes` is expected to be sorted in the first place.

File: packages/gatsby-source-contentful/src/extend-node-type.js [hunk 3]
```diff
@@ -286,19 +281,17 @@ const resolveFluid = (image, options) => {
 
   // Add the original image (if it isn't already in there) to ensure the largest image possible
   // is available for small images.
+  const pwidth = parseInt(width, 10)
   if (
-    !filteredSizes.includes(parseInt(width)) &&
-    parseInt(width) < CONTENTFUL_IMAGE_MAX_SIZE &&
-    Math.round(width / desiredAspectRatio) < CONTENTFUL_IMAGE_MAX_SIZE
+    !filteredSizes.includes(parseInt(pwidth, 10)) &&
```
Comment (vladar) [inline packages/gatsby-source-contentful/src/extend-node-type.js:286]:
```suggestion
    !filteredSizes.includes(pwidth) &&
```

File: packages/gatsby-source-contentful/src/extend-node-type.js (hunk 1 above)
Comment (pvdz) [reply packages/gatsby-source-contentful/src/extend-node-type.js:182]: Sure

File: packages/gatsby-source-contentful/src/extend-node-type.js [hunk 4]
```diff
@@ -178,11 +176,8 @@ const resolveFixed = (image, options) => {
     )
   })
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
-
   // Create the srcSet.
```
Comment (pvdz) [inline packages/gatsby-source-contentful/src/extend-node-type.js:179]:
```suggestion
  // Create the srcSet. The `filteredSizes` array is assumed to be sorted asc.
```

File: packages/gatsby-source-contentful/src/extend-node-type.js [hunk 5]
```diff
@@ -286,19 +281,17 @@ const resolveFluid = (image, options) => {
...
+    Math.round(pwidth / desiredAspectRatio) < CONTENTFUL_IMAGE_MAX_SIZE
   ) {
-    filteredSizes.push(width)
+    filteredSizes.push(pwidth)
   }
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
-
   // Create the srcSet.
```
Comment (pvdz) [inline packages/gatsby-source-contentful/src/extend-node-type.js:293]:
```suggestion
  // Create the srcSet. The `filteredSizes` array is assumed to be sorted asc.
```

**Comment by pvdz** [comment]
Ok. This branch switching is a little funky :/

**Review by vladar — APPROVED** [review]
Looks good to me 👍 

P.S. Love the branch name 😄

**Pull request merged**